shotgun_api3/shotgun.py

```diff
@@ -2164,6 +2176,27 @@ def _make_call(self, verb, path, body, headers):
             attempt += 1
             try:
                 return self._http_request(verb, path, body, req_headers)
+            except SSLHandshakeError, e:
+                # Not a SHA-2 error or SHA-2 error we want to enforce. SHA-2 errors look like: 
+                #
+                #   [Errno 1] _ssl.c:480: error:0D0C50A1:asn1 encoding routines:ASN1_item_verify:
+                #   unknown message digest algorithm
+                #
+                # valid values should be "0" and "1" but we're trying to catch edge cases too
+                if not str(e).endswith("unknown message digest algorithm") or \
```

Comment:
Should test if we've already turned off validation so we don't end up warning three times if things keep failing with certs disabled.